mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactTextView.js

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { getSrc } from './ShowArtifactPage';
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { coy as style } from 'react-syntax-highlighter/dist/esm/styles/prism';
```

Comment:
This page compares the themes `react-syntax-highlighter` supports:

https://k8shiro.github.io/ReactCompareCodeHighlighter/